src/components/portifolio-header/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./style.css";
+
+export default function Header() {
```

Comment:
Sugiro alterar o nome do componente para `PortfolioHeader`
```suggestion
export default function PortfolioHeader() {
```